sys/vane/gall.hoon

```diff
@@ -1241,11 +1243,14 @@
         $serv  `%e
         $tend  `%a
         $them  `%e
+        $turf  `%j
+        $vein  `%j
```

Comment:
@philipcmonk, were these two just for testing? I expect the other additions here to be used from userspace, but these two seem less likely. (And sorry I missed this in #870).

Reply:
Yeah, you're right, I'll take those out